### DISCUSSION TITLE
Change Update interval if Moving is detected

### DISCUSSION THREAD

**Discussion by kurim** [flobz/psa_car_controller#269] [Ideas]
Is it possible to implement a function if psa car controller detect moving state to decrease -R 5 to 1 for example to get a more accurate route?

Since as soon car moves it change to: moving:True we can decrease and as soon as moving:False we can increase the value back to normal, may be with a delay to avoid stops updating like stand at a traffic light or something.

**Comment by flobz** [comment]
The simple solution is to set it to 2.
It's the maximum rate allowed by the API.
Furthermore I don't think that the car send its position as often.

**Reply by kurim** [reply]
Hm some how it must be send very often, or how does tronity generate a very accurate map
![image](https://user-images.githubusercontent.com/1880220/141091738-274d8592-5d82-4c33-b0b8-37479cbef486.png)
Or did they just store more positions, not just start and end?

Btw. it update very frequently:
```log
longitude:x.90591 latitude:xx.16497 date:2021-11-10 13:54:16+00:00
longitude:x.91623 latitude:xx.1643 date:2021-11-10 13:55:46+00:00
```

So the position already changed after 30second
